config/301-ibmmqtarget.yaml

```
@@ -0,0 +1,203 @@
+# Copyright 2021 TriggerMesh Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: ibmmqtargets.targets.triggermesh.io
+  labels:
+    duck.knative.dev/addressable: 'true'
+    triggermesh.io/crd-install: 'true'
+  annotations:
+    registry.triggermesh.io/acceptedEventTypes: |
+      [
+        { "type": "*" }
+      ]
+    registry.knative.dev/eventTypes: |
+      [
+        { "type": "io.triggermesh.ibmmqtarget.response" }
+      ]
+spec:
+  group: targets.triggermesh.io
+  names:
+    kind: IBMMQTarget
+    plural: ibmmqtargets
+    categories:
+    - all
+    - knative
+    - eventing
+    - targets
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+    subresources:
+      status: {}
+    schema:
+      openAPIV3Schema:
+        type: object
+        description: TriggerMesh event target for IBM MQ.
+        properties:
+          spec:
+            description: Desired state of the event target.
+            type: object
+            properties:
+              connectionName:
+                description: IBM MQ server URI, e.g. ibm-mq.default.svc.cluster.local(1414).
+                type: string
+              channelName:
+                description: The name of IBM MQ Channel to listen events on.
+                type: string
+              queueManager:
+                description: IBM MQ Queue Manager name.
+                type: string
+              queueName:
+                description: IBM MQ event source Queue name.
```

im pretty shur i know what your asking for... but i think this description could be confusing to a lot of users. 
 Maybe try:
 
 ```suggestion
                description: The IBM MQ Queue name. 
```